config/default.yml

```diff
@@ -2055,8 +2055,9 @@ Lint/SuppressedException:
   StyleGuide: '#dont-hide-exceptions'
   Enabled: true
   AllowComments: true
+  AllowRescueNil: true
```

Comment:
I think `AllowNil` would be a better name here.

Reply:
Makes sense! Will fix it!